tests/e2e/unified_test.go

```diff
@@ -171,6 +171,10 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 					create.MaybeSkip(t, fixture.Name, opt.Create)
 				}
 
+				if testVCR {
+					os.Setenv("VCR_PATH", fixture.SourceDir)
```

Comment:
I think I would prefer to pass this in to the harness via a field or method parameter (because env vars will break in parallel and are unobvious i.e. are invisible to static analysis).  If we need it in the constructor (`NewHarness`), then I propose creating a `HarnessOptions` struct.

Reply:
Done